core/urls.py

```diff
@@ -11,4 +11,5 @@
     url(r'^grappelli/', include('grappelli.urls')),
     url(r'^admin/', admin.site.urls),
     url(r'^api$', csrf_exempt(authorization_required(GraphQLView.as_view()))),
+    url(r'^gh$', csrf_exempt(GraphQLView.as_view(graphiql=True))),
```

Comment:
`graphiql=True` is for enabling the webui which I don't think is useful as the api_key needs to set in the header to make any request.